pkg/kubelet/volumes.go

```diff
@@ -111,6 +104,9 @@ func (vh *volumeHost) GetHostName() string {
 	return vh.kubelet.hostname
 }
 
+// mountExternalVolumes creates a VolumeMap and attempts to physically mount
```

Comment:
strike this comment, I made another PR adding a comment for this method (there's a lot more to say about it than this)